modules/appeals_api/app/swagger/appeals_api/v2/swagger.json

```diff
@@ -7026,6 +7047,26 @@
                           "email": "bobsemail@bobbytablesemail.com",
                           "timezone": "America/Chicago"
                         },
+                        "claimant": {
```

Comment:
Same here. These are fine to put into our swagger_dev.json, but isn't quite ready for prod yet.